api/HTMLCanvasElement.json

```
@@ -1018,6 +1018,38 @@
           }
         }
       },
+      "mozPrintCallback": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "18"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
```

It's just in one engine, right?

```suggestion
            "experimental": true,
```

---

yes, this is a firefox-only feature

but I think it is not required to mark it as experimental, as it has been released since 2012

---

Ah yes, I forgot about this rule. Thank you!